### PR TITLE
Process GCP persistent disks with embedded raw image

### PR DESCRIPTION
**Pull request by aarontp**
Note:  Also needed to fully support this type of evidence format is a task to do the conversion from raw image files in GCS to GCP PD with embedded raw images (Issue #63 ).

### CODE REVIEW

File: turbinia/processors/mount_local.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+from turbinia import config
+from turbinia import TurbiniaException
+
+log = logging.getLogger('turbinia')
+
+def PreprocessMountDisk(evidence):
+  """Locally mounts disk in an instance.
+
+  Args:
+    evidence: A turbinia.evidence.RawDisk or subclass object.
```
Comment (berggren) [inline turbinia/processors/mount_local.py]: This is a bit unclear to me. Subclass of what?

File: turbiniactl [hunk 2]
```diff
@@ -108,6 +108,31 @@ if __name__ == '__main__':
...
+      required=True)
+  parser_googleclouddiskembedded.add_argument(
+      '-s',
+      '--source',
+      help='Description of the source of the evidence',
+      required=False)
+  parser_googleclouddiskembedded.add_argument(
+      '-n', '--name', help='Descriptive name of the evidence', required=False)
+
   # Directory
```
Comment (berggren) [inline turbiniactl]: This comment doesn't add much. Maybe reword it to be more descriptive?

File: turbinia/processors/mount_local.py (hunk 1 above)
Comment (aarontp) [reply turbinia/processors/mount_local.py]: Updated to make it a little more clear.

File: turbiniactl (hunk 2 above)
Comment (aarontp) [reply turbiniactl]: Reworded.

**Comment by aarontp** [comment]
Thanks!
